WORKSPACE

```diff
@@ -38,17 +38,21 @@ bind(
 # 2. Update .bazelversion, envoy.bazelrc and .bazelrc if needed.
 #
 # envoy-wasm commit date: Jan 23 2020
```

Comment:
change this to istio/envoy?

Reply:
The commit date is for the envoy-wasm fork into istio/envoy after that it is just cherry-picks, so I was going to leave it, but perhaps istio/envoy would be better if less informative in some ways. WDYT?

Reply:
yeah otherwise,  wasm here seems very confusing..